models/__init__.py

```diff
@@ -0,0 +1,279 @@
+# coding=utf-8
+import pandas as pd
```

Comment:
I may not be leaving pandas in this unless it brings a very specific advantage or feature. there are some test/learning functions towards the bottom of this file where i'm pulling joined table data directly to build some dataframes, but its unlikely to provide enough advantage to keep the library.